frontend/packages/kubevirt-plugin/src/components/vm-disks/create-disk-row.tsx

```diff
@@ -93,8 +93,16 @@ export const CreateDiskRow: React.FC<CreateDiskRowProps> = ({
             nameError && nameError.type === ValidationErrorType.Error ? nameError.type : null
           }
         >
-          <Text id="disk-name" disabled={creating} onChange={setName} value={name} />
-          <HelpBlock className="kubevirt-vm-create-device-row__cell--help">
+          <Text
+            id="disk-name"
+            disabled={creating}
+            onChange={(v) => {
+              setName(v);
+              forceRerender();
```

Comment:
Why does change of state not lead to automatic re-render by React?
The flow is already complicated. I am afraid that this low-level play will increase complexity even more.

Reply:
> Why does change of state not lead to automatic re-render by React?

Because rendering is custom handled by `componentDidUpdate` in virtualized `Table`. 

> The flow is already complicated. I am afraid that this low-level play will increase complexity even more.

IMO it is okay for the time being, because this feature will be removed very soon (replaced by dialog).

It prevents UI bugs where the name error is rendered over the following rows

Reply:
ok, got it. As this code is about to be replaced, we can keep it.